jalien-xrootd.sh

```diff
@@ -0,0 +1,59 @@
+package: JAlien-XRootD
```

Comment:
Do not make a separate package. We have a recipe for that. We can use a different set of defaults for testing and make a single PR out of this one and #1084.